src/reporters/slack.js

```diff
@@ -66,7 +70,28 @@ const createSection = ( text, type = 'mrkdwn' ) => {
     };
 };
 
-const getMessage = ( { name, block, error } ) => {
+const getThreadMessage = () => {
+    const message = [];
+
+    let sectionMessage = `*Travis build:* <${ TRAVIS_BUILD_WEB_URL }|#${ TRAVIS_JOB_NUMBER }>
```

Comment:
We could use `TRAVIS_JOB_WEB_URL` here instead as these threads will be job specific.